Neumann2023

DianaSpurite opened the pull request:
# Pull Request

This PR is for a

- [X] [New Publication(s)](#new-publication)
- [ ] [Correction](#correction)

For the following list(s):

- [ ] ancientmetagenome-environmental ([README](https://github.com/SPAAM-workshop/AncientMetagenomeDir/tree/master/ancientmetagenome-environmental))
- [ ] ancientmetagenome-hostassociated ([README](https://github.com/SPAAM-workshop/AncientMetagenomeDir/tree/master/ancientmetagenome-hostassociated))
- [X] ancientsinglegenome-hostassociated ([README](https://github.com/SPAAM-workshop/AncientMetagenomeDir/tree/master/ancientsinglegenome-hostassociated))

## New Publication

### Publication Information

This pull request is to add samples from the following publication(s): https://austriaca.at/bronzezeit-pest-in-drasenhofen

This is to close #1092

### Checklist

- [x] Publication is published (preprints currently not accepted)?
- [x] Checked the publication is not already in the database?
- [x] Checked samples in this publication are not previously published _data_ (newly re-sequenced metagenomes are OK!)?
- [x] Samples are shotgun metagenomes (hostassociated-singlegenome may also contain whole-genome enriched data)?
- [x] Checked the list follows conventions as described in the corresponding sample type's README file (e.g. using ERS/SRS accession codes for ENA/SRA)?
- [x] Changelog is updated to include the publication under 'Added'?
- [x] Pull request has passed validation checks (see automated comment from `github-bot`)?
- [x] Review requested from `@SPAAM-community/ancientmetagenomedir`?
- [x] (If applicable) Made a separate PR to add new categories to controlled lists (stored under `/assets/enums`, e.g. material or archive)? If so the corresponding PR is here: <!-- REPLACE this comment  (including the < > symbols) with that PR -->

DianaSpurite commented:
1. library_polymerase as always in a different dimension
2. also didn't find a doi

jfy133 commented on CHANGELOG.md:
```diff
@@ -32,6 +32,7 @@ and this project follows to [Calendar Versioning](https://calver.org/).
 #### Ancient Single Genome: Host Associated
 
 - Swali 2023 10.1038/s41467-023-38393-w (added by @DianaSpurite)
+- Neumann 2023 austriaca.at/bronzezeit-pest-in-drasenhofen (added by @DianaSpurite)
```
```suggestion
- Neumann 2023 10.1553/archaeologia107 (added by @DianaSpurite)
```

jfy133 commented on ancientsinglegenome-hostassociated/samples/ancientsinglegenome-hostassociated_samples.tsv:
```diff
@@ -585,4 +585,6 @@ Sarhan2023	2023	10.1186/s12915-022-01509-7	Barfüsserkirche/Franciscan church of
 Swali2023	2023	10.1038/s41467-023-38393-w	Charterhouse Warren, Mendip Hills, Somerset	51.299	-2.715	United Kingdom	C10091	Homo sapiens	4000	10.1038/s41467-023-38393-w	bacteria	Yersinia pestis	tooth	chromosome	ENA	raw	PRJEB61230	ERS14891340
 Swali2023	2023	10.1038/s41467-023-38393-w	Charterhouse Warren, Mendip Hills, Somerset	51.299	-2.715	United Kingdom	C10098	Homo sapiens	4000	10.1038/s41467-023-38393-w	bacteria	Yersinia pestis	tooth	chromosome	ENA	raw	PRJEB61230	ERS14891341,ERS14891342
 Swali2023	2023	10.1038/s41467-023-38393-w	Levens Deer Park ring cairn, Leven, Cumbria	54.268	-2.761	United Kingdom	C10928	Homo sapiens	3900	10.1038/s41467-023-38393-w	bacteria	Yersinia pestis	tooth	chromosome	ENA	raw	PRJEB61230	ERS14909714,ERS14891343
+Neumann2023	2023	10.1553/archaeologia107	Cemetery of Drasenhofen	48.755	16.643	Austria	DSH008	Homo sapiens	4000	10.1553/archaeologia107	bacteria	Yersinia pestis	tooth	chromosome	ENA	raw	PRJEB61070	ERS15547237
+Neumann2023	2023	10.1553/archaeologia107	Cemetery of Drasenhofen	48.755	16.643	Austria	DSH025	Homo sapiens	4000	10.1553/archaeologia107	bacteria	Yersinia pestis	tooth	chromosome	ENA	raw	PRJEB61070	ERS15547238
```
I think you're still missing 'ERS15547236' and 'ERS15547235' (even if the YP capture date was the only thing used in teh final genome (I assume), shotgun data is valid to include in here :) ), becuase technically sg/capture _should not_ have separate ERS codes!

jfy133 reviewed (changes requested):
Library table has the wrong ERS codes assoigned with the sample IDS (e.g. ERS ending in 37 should be with DSH008)

jfy133 commented on ancientsinglegenome-hostassociated/samples/ancientsinglegenome-hostassociated_samples.tsv:
```diff
@@ -585,4 +585,6 @@ Sarhan2023	2023	10.1186/s12915-022-01509-7	Barfüsserkirche/Franciscan church of
 Swali2023	2023	10.1038/s41467-023-38393-w	Levens Deer Park ring cairn, Leven, Cumbria	54.268	-2.761	United Kingdom	C10928	Homo sapiens	3900	10.1038/s41467-023-38393-w	bacteria	Yersinia pestis	tooth	chromosome	ENA	raw	PRJEB61230	ERS14909714,ERS14891343
+Neumann2023	2023	10.1553/archaeologia107	Cemetery of Drasenhofen	48.755	16.643	Austria	DSH008	Homo sapiens	4000	10.1553/archaeologia107	bacteria	Yersinia pestis	tooth	chromosome	ENA	raw	PRJEB61070	ERS15547237,ERS15547235
+Neumann2023	2023	10.1553/archaeologia107	Cemetery of Drasenhofen	48.755	16.643	Austria	DSH025	Homo sapiens	4000	10.1553/archaeologia107	bacteria	Yersinia pestis	tooth	chromosome	ENA	raw	PRJEB61070	ERS15547238,ERS15547236
 
```
```suggestion
```

jfy133 commented:
Bug was due to https://github.com/SPAAM-community/AMDirT/issues/125, I've pinned the JSONschema package for you and will keep that temporarily now

jfy133 reviewed (changes requested):
> und in eine sequencing library transformiert.12 - > 12 Gansauge et al. 2020. -> Gansauge et al. 2020
M.-T. Gansauge, A. Aximu-Petri, S. Nagel, M. Meyer, Manual
and automated preparation of single-stranded DNA libraries
for the sequencing of DNA from ancient biological remains and
other sources of highly degraded DNA,

Single stranded for the shotgun sequence liraries (2 missing from library list!), and this was with no UDG treatmenment

Also the Gansauge is also cited for the capture libraries so they should also be singlestranded not doublre

Polymerase for Gansauge 2020 is: AccuPrime Pfx DNA 

But other than that looks OK (@GUNeumann is also reviewing though :D)

GUNeumann commented on ancientsinglegenome-hostassociated/libraries/ancientsinglegenome-hostassociated_libraries.tsv:
please, change:
strand type: "single"
library_polymerase: "AccuPrime Pfx DNA"
library_treatment: "none" for DSH008.A0101 and DSH025.A0101 "half-udg" for DSH008.A0201 and DSH025.A0201

the rest looks fine! :)

DianaSpurite commented:
> Single stranded for the shotgun sequence liraries (2 missing from library list!), and this was with no UDG treatmenment
Added the two libraries

> Also the Gansauge is also cited for the capture libraries so they should also be singlestranded not doublre
changed

> Polymerase for Gansauge 2020 is: AccuPrime Pfx DNA
added

jfy133 commented:
I approve on behalf of @GUNeumann as his requested changes overlapped with mine and/or has been addressed as requested
